client/components/App.jsx

```diff
@@ -1,19 +1,50 @@
 import React from 'react';
-
+import ReviewList from './ReviewList.jsx';
 
 class App extends React.Component {
   constructor() {
     super();
     this.state = {
+      comments: [],
+      ratings: 0,
+      reviews: 0,
+      seller: '',
 
     };
   }
 
+  componentDidMount() {
+    fetch('/api/5da6350a92b86a1332313b4c')
```

Comment:
You should reassign a shorter id or make sure that your endpoint is understandable at first glance. Something like /api/comments